<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NTINLTP2001429" name="Default test" tests="7" failures="1" timestamp="14 Aug 2018 09:15:16 GMT" time="9.529" errors="0">
  <testcase name="FailTest" time="8.675" classname="extentReportPkg.exTentReportsClass">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[https://www.google.co.in/]&amp;gt; but was:&amp;lt;[NoTitle]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[https://www.google.co.in/]> but was:<[NoTitle]>
	at org.junit.Assert.assertEquals(Assert.java:125)
	at org.junit.Assert.assertEquals(Assert.java:147)
	at extentReportPkg.exTentReportsClass.FailTest(exTentReportsClass.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- FailTest -->
  <testcase name="FailTest1" time="0.001" classname="extentReportPkg.exTentReportsClass"/>
  <testcase name="passTest" time="0.001" classname="extentReportPkg.exTentReportsClass"/>
  <testcase name="passTest1" time="0.001" classname="extentReportPkg.exTentReportsClass"/>
  <testcase name="passTest2" time="0.002" classname="extentReportPkg.exTentReportsClass"/>
  <testcase name="skipTest" time="0.001" classname="extentReportPkg.exTentReportsClass">
    <skipped/>
  </testcase> <!-- skipTest -->
  <testcase name="skipTest1" time="0.013" classname="extentReportPkg.exTentReportsClass">
    <skipped/>
  </testcase> <!-- skipTest1 -->
</testsuite> <!-- Default test -->
